apps/data/pgsync/schema.json

```diff
@@ -0,0 +1,49 @@
+[
+    {
+        "database": "martletplace",
+        "nodes": {
+            "table": "listings",
+            "columns": [
+                "listing_id",
+                "seller_id",
+                "title",
+                "price",
+                "location",
+                "status",
+                "description",
+                "image_urls",
+                "created_at"
+            ],
+            "children": [
+                {
+                    "table": "users",
+                    "columns": [
+                        "name"
+                    ],
+                    "relationship":
+                    {
+                        "variant": "object",
+                        "type": "one_to_one",
+                        "foreign_key": {
+                            "child": [
+                                "user_id"
+                            ],
+                            "parent": [
+                                "seller_id"
+                            ]
+                        }
+                    }
+                }
+            ],
+            "transform":
+            {
+                "rename":
```

Comment:
I'm assuming we can use another rename to change from latitude -> lat, etc. Sounds like a problem for later

Reply:
I tried everything I could find in the PGSync docs to do this. I think it PGSync doesn't properly support Postgres type definitions so it can't do the rename as part of the sync operation.